system-parachains/people/people-polkadot/src/bridge_to_bulletin_config.rs

```diff
@@ -0,0 +1,280 @@
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Bridge definitions used on PeopleHubPolkadot for bridging to Polkadot Bulletin.
+//!
+//! Polkadot Bulletin chain will be the 1:1 copy of the Polkadot Bulletin, so we
+//! are reusing Polkadot Bulletin chain primitives everywhere here.
+
+use crate::{
+	bridge_common_config::RelayersForLegacyLaneIdsMessagesInstance,
+	xcm_config::UniversalLocation, AccountId, Balance, Balances, BridgePolkadotBulletinGrandpa,
+	BridgePolkadotBulletinMessages, Runtime, RuntimeEvent, RuntimeHoldReason, XcmOverPolkadotBulletin,
+	XcmRouter,
+};
+use bp_messages::{
+	source_chain::FromBridgedChainMessagesDeliveryProof, target_chain::FromBridgedChainMessagesProof, LegacyLaneId,
+};
+
+use frame_support::{
+	parameter_types,
+	traits::{Equals, PalletInfoAccess},
+};
+use frame_system::{EnsureNever, EnsureRoot};
+use pallet_bridge_messages::LaneIdOf;
+use pallet_bridge_relayers::extension::{
+	BridgeRelayersTransactionExtension, WithMessagesExtensionConfig,
+};
+use pallet_xcm_bridge_hub::XcmAsPlainPayload;
+use polkadot_parachain_primitives::primitives::Sibling;
+use system_parachains_constants::polkadot::currency::UNITS as ROC;
+use xcm::{
+	latest::prelude::*,
+	prelude::{InteriorLocation, NetworkId},
+	AlwaysV5,
+};
+use xcm_builder::{BridgeBlobDispatcher, ParentIsPreset, SiblingParachainConvertsVia};
+
+parameter_types! {
+	/// Interior location (relative to this runtime) of the with-PolkadotBulletin messages pallet.
+	pub BridgePolkadotToPolkadotBulletinMessagesPalletInstance: InteriorLocation = [
+		PalletInstance(<BridgePolkadotBulletinMessages as PalletInfoAccess>::index() as u8)
+	].into();
+	/// Polkadot Bulletin Network identifier.
+	pub PolkadotBulletinGlobalConsensusNetwork: NetworkId = NetworkId::PolkadotBulletin;
+	/// Relative location of the Polkadot Bulletin chain.
+	pub PolkadotBulletinGlobalConsensusNetworkLocation: Location = Location::new(
+		2,
+		[GlobalConsensus(PolkadotBulletinGlobalConsensusNetwork::get())]
+	);
+
+	// see the `FEE_BOOST_PER_RELAY_HEADER` constant get the meaning of this value
+	pub PriorityBoostPerRelayHeader: u64 = 58_014_163_614_163;
+
+	/// Priority boost that the registered relayer receives for every additional message in the message
+	/// delivery transaction.
+	///
+	/// It is determined semi-automatically - see `FEE_BOOST_PER_MESSAGE` constant to get the
+	/// meaning of this value.
+	pub PriorityBoostPerMessage: u64 = 364_088_888_888_888;
+
+	/// PeoplePolkadot location
+	pub PeoplePolkadotLocation: Location = Location::new(1, [Parachain(polkadot_runtime_constants::system_parachain::PEOPLE_ID)]);
```

Comment:
hmm, this is no good, it should be `Here` at least, but let's revisit this with `BridgeOriginAccountIdConverter` TODO - maybe we don't even need this
```suggestion
	pub PeoplePolkadotLocation: Location = Here.into_location();
```